### PR TITLE
Move logout button

### PR DESCRIPTION
**Pull request by wrunion**
I wrote this in the convo too, but I moved the logout button into the Header component, so it's now on top. I don't know SASS yet though, and I couldn't get it to the right. 

It seems it's where it needs to be now though, component-wise, and I made a note of where the new CSS rules need to go. I'll read up on Burma and SASS, but I wanted to get this request moving along, and see if anyone else knows which rules are affecting which components, and how to override them for this.

Alright!

### CODE REVIEW

**Review by chrisandrewca — DISMISSED** [review]
Looks great!

**Comment by Chris-Boe** [comment]
It looks like you started this branch off your old `winter` branch, which still had the changes to the /stashed folder. While it's no big deal here, remember to always start off from development when you can!

File: src/scss/helpers.scss [hunk 1]
```diff
@@ -30,8 +30,7 @@
 }
 
 .is-logout-button{
-	padding-top: 796px;
-  padding-left: 20px;
+  /* TO DO: create a rule to move button to the top right */
```
Comment (Chris-Boe) [inline src/scss/helpers.scss:33]: Let me know if you need help with the CSS for this! I have an example

**Review by Chris-Boe — CHANGES_REQUESTED** [review]

**Comment by Chris-Boe** [comment]
Thanks for your work Winter, Darren got this finished up while you are away.
Hope to see you again soon and hope you are staying healthy and safe!

**Pull request closed without merging**